### DISCUSSION TITLE
Incomplete Transaction List...

### DISCUSSION THREAD

**Discussion by lukewealth** [aptos-labs/aptos-developer-discussions#120] [Questions]
Hey all please i am having an issue with the aptos ts sdk. 
This method seems to return an incomplete transaction list... cant fint my latest deposit events, only withdrawals.... Can someone help me out. I have been struggling for weeks.

![image](https://github.com/aptos-labs/aptos-developer-discussions/assets/22635598/7067ee99-05dc-457c-ac06-00303b71dc4b)

also what will be the best way to get the wallet transactions. I am building a mini wallet and trying to get transaction history and nft list
thaks in advance.

**Comment by 0x-j** [comment]
Can you give me an address you called with? When you say it doesn't return complete transactions, do you mean you see some txs on explorer but not in the response?

**Reply by Ovodo** [reply]
Yes exactly....
This is the address
0x30fc5066aa21bdf9d2ab60353a81601927ea2877966adea38ae821f55b976891

**Reply by 0x-j** [reply]
Ok turns out `getAccountTransactions` only returns txs sent by you. It doesn't return txs that you are involved (e.g. someone else sent a tx that transfers some token to you). I created an issue in our ts sdk. Hopefully we can implement this soon.

https://github.com/aptos-labs/aptos-ts-sdk/issues/352

**Reply by 0x-j** [reply]
Thanks for reporting this, we should definitely fix this ASAP, it's very confusing to see different result between sdk and explorer.

**Reply by Ovodo** [reply]
Okay thank you..is there a way I can get the correct details from the explorer in the mean time

**Reply by 0x-j** [reply]
You can try to copy the [logic in explorer](https://github.com/aptos-labs/explorer/blob/main/src/pages/Account/Tabs/TransactionsTab.tsx#L23), I'll try to implement it in SDK on Monday and will let you know ASAP.
